features/configuration/color.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Windows may require additional solutions to display color
+
+  The output uses ANSI escapte codes to show text in color.  Windows
```

Comment:
Also, do you think it's worth turning "ANSI escape codes" into a link to https://en.wikipedia.org/wiki/ANSI_escape_code ?